app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -75,15 +75,17 @@ def pvm_instances
       instance.volume_i_ds.to_a.each do |vol_id|
         volume = collector.volume(vol_id)
 
-        persister.disks.build(
-          :hardware        => ps_hw,
-          :device_name     => volume.name,
-          :device_type     => volume.disk_type,
-          :controller_type => "ibm",
-          :backing         => persister.cloud_volumes.find(vol_id),
-          :location        => vol_id,
-          :size            => volume.size&.gigabytes
-        )
+        unless volume.nil?
```

Comment:
Since @miq-bot doesn't approve of this, instead I could do:
```suggestion
        next if volume.nil?
```

Reply:
:+1: yes that is the preferred style